src/components/V4HomePage.js

```diff
@@ -92,9 +97,15 @@ const HomepageWrapper = ({
           />
           <Helmet>
             <link rel="canonical" href={canonicalURL} />
-            {referrer ? <meta name="robots" content="noindex" /> : null}
+            {referrer || isPreviewPage ? (
+              <meta name="robots" content="noindex" />
+            ) : null}
           </Helmet>
-          <Landing landingData={landing} causeId={causeId} />
+          <Landing
+            landingData={landing}
+            causeId={causeId}
+            previewMessage={isPreviewPage}
+          />
```

Comment:
For UI testing, could you add a Chromatic variant with the preview enabled (probably just for the `Landing` component)?